.circleci/config.yml

```diff
@@ -238,6 +247,8 @@ jobs:
           key: dependency-cache-v1-{{ checksum "yarn.lock" }}
           paths:
             - node_modules/
```

Comment:
Might be better just to cache the `cache` directory, not `node_modules`. This really bloats the size, and makes this cache no longer safe to share between Node.js versions because of differences in postinstall steps for certain dependencies.

Reply:
Ill remove it. I tried both. The link step takes 20s longer without the node_modules cached but thats not too bad.